man/mkdocs/grassdocs.css

```diff
@@ -1,10 +1,129 @@
 :root > * {
-  --md-primary-fg-color:        #088B36;
-  --md-primary-fg-color--light: #088B36;
-  --md-primary-fg-color--dark:  #088B36;
-  --md-footer-bg-color:         #088B36;
-  --md-footer-bg-color--light:  #088B36;
-  --md-footer-bg-color--dark:   #088B36;
+
+  /* GRASS Variables */
+  /* ----------------------------------------------------------------------------- */
+  --primary-color: rgba(76, 176, 91, 1); /* Dark green */
+  --primary-color--light: rgba(158, 210, 156, 1); /* Light green */
+  --primary-color--dark: rgba(8, 139, 54, 1); /* Dark green */
+  --primary-bg-color: rgb(255, 255, 255); /* White */
+  --primary-bg-color--dark: rgba(135, 135, 135, 0.1);
+  --support-color: rgb(243, 57, 138); /* Pink */
+
+  /* MkDocs Variables */
+  /* ----------------------------------------------------------------------------- */
+
+  /* Default Colors */
+  /* ----------------------------------------------------------------------------- */
+  --md-default-fg-weight: 800;
+
+  /* Primary Colors */
+  /* ----------------------------------------------------------------------------- */
+  --md-primary-fg-color:        var(--primary-color); /* Dark green */
+  --md-primary-fg-color--light: rgb(255, 255, 255);
+  --md-primary-fg-color--dark:  var(--primary-color--dark);
+  --md-primary-bg-color:        rgb(255, 255, 255); /* Includes color for the navbar text */
+  --md-primary-bg-color--light: rgba(255, 255, 255, 0.5);
```

Comment:
For this and below, please provide info which would help updating this in the future. (Is there a doc or command to generate current list of variables, for example?)

Reply:
I've added links to docs and some instructions as comments in the file.

Reply:
The comments look good. Thanks.